rpms/CentOS/5/i386/Vagrantfile

```diff
@@ -0,0 +1,21 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "BraulioV/CentOS-5-i386"

```

Comment:
Please find below the documentation link for a more appropriate box configuration: 
https://www.vagrantup.com/docs/vagrantfile/machine_settings.html#config-vm-box_url

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/104/commits/4aab9d1a4ae844ffeb991db6459d9e58f0403e82.